CMakeLists.txt

```diff
@@ -193,6 +195,11 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
           PROPERTIES
           COMPILE_FLAGS
           "-gencode arch=compute_90a,code=sm_90a")
+  set_source_files_properties(
+        "csrc/quantization/cutlass_w8a8/scaled_mm_dq_bias_c3x.cu"
+        PROPERTIES
+        COMPILE_FLAGS
+        "-gencode arch=compute_90a,code=sm_90a")
```

Comment:
You should be able to do something like:
```  
list(APPEND HOPPER_KERNELS
  "csrc/quantization/cutlass_w8a8/scaled_mm_dq_c3x.cu"
  "csrc/quantization/cutlass_w8a8/scaled_mm_dq_bias_c3x.cu"
)

set_source_files_properties(
    ${HOPPER_KERNELS}
    PROPERTIES
    COMPILE_FLAGS
    "-gencode arch=compute_90a,code=sm_90a")
```

Reply:
incorporated